outsource/meta.py

```diff
@@ -0,0 +1,47 @@
+# These data_types need per-chunk storage, so don't upload to rucio here!
+PER_CHUNK_DATA_TYPES = [
+    "peaklets",
```

Comment:
Why is records removed?

Reply:
`records` are not always saved by straxen: https://github.com/XENONnT/straxen/blob/ff7c787e7769cfc05192c96a3b19c51eaedc7df1/straxen/plugins/records/records.py#L44. So I disallow records being in include_data_types. The processing of `records` will only be a temporary step of the processing of `peaklets`.